src/store/getters/state-getters.ts

```diff
@@ -428,7 +433,8 @@ export const hasAlterationDataChanged = (state: StateIF): boolean => {
 export const hasSpecialResolutionDataChanged = (state: StateIF): boolean => {
   return (
     hasBusinessNameChanged(state) ||
-    hasBusinessTypeChanged(state)
+    hasBusinessTypeChanged(state) ||
+    hasAssociationTypeChanged(state)
```

Comment:
Need address check here?

Is there also a "is valid" getter for SR?

Reply:
Addresses can't be changed, still need an address check?

Reply:
I'll put in a isValid for SR

Reply:
If addresses can't be changed then there's no need to check if they've changed or if they make the filing valid.

I think it's worth a comment in `hasSpecialResolutionDataChanged()` saying addresses are not included, and maybe in your "is valid" getter.

Reply:
Looks good.